stable/rabbitmq/values-production.yaml

```diff
@@ -292,6 +292,7 @@ metrics:
     ##
     # pullSecrets:
     #   - myRegistryKeySecretName
+
```

Comment:
Please either remove this blank line or also add it to the values-production.yaml

Reply:
they are now realigned